finagle-http/src/main/scala/com/twitter/finagle/http/codec/ConnectionManager.scala

```diff
@@ -11,7 +11,7 @@ import com.twitter.util.{Future, Promise}
 private[finagle] class ConnectionManager {
   private[this] var isKeepAlive = false
   private[this] var isIdle = true
-  private[this] var activeStreams = 0
+  private[this] var activeStreams, pendingResponses = 0
```

Comment:
@mosesn I don't quite understand... `pendingResponses` is always incremented in `observeRequest` and always decremented in `observeResponse`.  Isn't `pendingResponses` is an accurate name? Is there a situation where a response is observed before a request?